Client/Frontend/Browser/Toolbars/BottomToolbar/Menu/VPNMenuButton.swift

```diff
@@ -69,7 +71,10 @@ struct VPNMenuButton: View {
     
     var body: some View {
         HStack {
-            MenuItemHeaderView(icon: #imageLiteral(resourceName: "vpn_menu_icon").template, title: "Brave VPN")
+            MenuItemHeaderView(
+                icon: #imageLiteral(resourceName: "vpn_menu_icon").template,
+                title: "Brave VPN",
```

Comment:
reuse localization

Reply:
See https://github.com/brave/brave-ios/pull/4424#discussion_r741417115

Reply:
No localization here "Brave VPN" is the exception for localization.